### ISSUE TITLE
Stability of country names and/or ISO codes

### ISSUE THREAD

**Issue by geotheory** [CSSEGISandData/COVID-19#616]
Awesome job everyone. Small request from users of your data: please can you ensure stability of country names?  I noticed "UK" changed yesterday to "United Kingdom".  This sort of thing is going to make life difficult for anyone linking records to other datasets.  Ideally the addition of a recognised code (e.g. _"ISO2C"_ [ISO 3166-1 alpha-2 ](https://en.wikipedia.org/wiki/ISO_3166-1_alpha-2)) would remove any ambiguity, but if not possible stable country names is critical :)

**Comment by oliverbooth** [comment]
Seconded, along with standardised ISO date/time format.

Date format in 01-22-2020.csv:
M/DD/YYYY HH:mm

Date format in 01-23-2020.csv:
M/DD/YY HH:mm

Date format in 02-02-2020.csv:
Only here from does it finally use ISO 8601

**Comment by emilianbold** [comment]
There's also: `St. Martin/Saint Martin`, `Taiwan/Taiwan*`, `Mainland China/China` and `Others/Cruise Ship`. All introduced with the yesterday update.

**Comment by AnthonyEbert** [comment]
https://github.com/AnthonyEbert/COVID-19_ISO-3166/blob/master/JohnsHopkins-to-A3.csv

**Comment by benmaier** [comment]
@CSSEGISandData please consider this.

**Comment by ablanch5** [comment]
I agree this is making it difficult to merge this data with data from other sources.

**Comment by pixelscript** [comment]
FYI I've got a little script here that'll turn the data into a consistent JSON set. Includes ISO 3166-1 alpha-3 and alpha-2 codes.

 https://github.com/pixelscript/covid-19-data
